spec/fixtures/examples/ead_component.xml

```diff
@@ -1,25 +0,0 @@
-<c id="ref12" level="item">
-    <did>
-        <unittitle>[ACT UP Paris]</unittitle>
-        <container id="cid17653001" type="Box" label="Mixed materials">1</container>
-        <container parent="cid17653001" type="Folder">1</container>
-        <unitdate normal="1992/1992">August 19, 1992</unitdate>
-        <origination label="creator">
-          <persname source="local">Bytsura, Bill</persname>
-          <persname source="naf">Belfrage, Sally, 1936-</persname>
-          <corpname rules="Anglo-American_Cataloguing_Rules__2nd_ed." source="naf">Kings County (N.Y.). Board of Supervisors.</corpname>
-          <famname source="naf">Lefferts family</famname>
-        </origination>
-        <controlaccess>
-          <genreform source="aat">Clippings (information artifacts)</genreform>
-          <genreform source="aat">Correspondence.</genreform>
-          <geogname source="lcsh">Cambridge (Mass.)</geogname>
-          <geogname source="lcsh">Spain |x History |y Civil War, 1936-1939 |x Atrocities.</geogname>
```

Comment:
@chrpr There was never an explicit test for that conversion, just implicit. I'm currently writing explicit tests for that as part of my next story.

You're correct that the boiler plate does not represent all our data, so we're just going to use it as a starting point and still have to augment the data on occasion.